communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "fastify.png",
+      "title": "Fastify",
+      "quote": "Fast and low overhead web framework, for Node.js",
```

Comment:
The `"quote"` is intended to be about your experience with Discord, would you mind writing a short blurb about your experience using Discord?